.pre-commit-config.json

```diff
@@ -1,40 +1,40 @@
 {
-  "repos": [
-    {
-      "hooks": [
-        {
-          "id": "trailing-whitespace"
-        },
-        {
-          "id": "end-of-file-fixer"
-        },
-        {
-          "id": "check-yaml"
-        },
-        {
-          "id": "check-added-large-files"
-        }
-      ],
-      "repo": "https://github.com/pre-commit/pre-commit-hooks",
-      "rev": "v3.2.0"
-    },
-    {
-      "hooks": [
-        {
-          "id": "golangci-lint"
-        },
-        {
-          "id": "go-fmt"
-        },
-        {
-          "id": "go-vet"
-        },
-        {
-          "id": "go-imports"
-        }
-      ],
-      "repo": "https://github.com/TekWizely/pre-commit-golang",
-      "rev": "master"
-    }
-  ]
+	"repos": [
```

Comment:
This file has to be updated according to the new hooks for rust, otherwise all non-nix contributors will not benefit of `pre-commit`.